src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -1543,8 +1542,8 @@ def braid2rels(L):
         P.SetTzOptions(dic)
         P.TzGoGo()
         P.TzGoGo()
-        gb = wrap_FpGroup(P.FpGroupPresentation())
-        U = [rel.Tietze() for rel in gb.relations()]
+        gb = P.FpGroupPresentation().sage()
+        U = [_.Tietze() for _ in gb.relations()]
```

Comment:
```suggestion
        U = [rel.Tietze() for rel in gb.relations()]
```
The `_` is generally meant for cases when the iterating object is not used in the iterator itself, e.g., `[1 for _ in range(5)]`.